[BUG] Fails to build with `dynamic-grammar-libs` feature since 0.7.0

```sh
$ cargo build --locked --no-default-features --features dynamic-grammar-libs
...
error[E0432]: unresolved import `crate::parse::supported_languages`
 --> src/main.rs:9:5
  |
9 | use crate::parse::supported_languages;
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `supported_languages` in `parse`

For more information about this error, try `rustc --explain E0432`.
error: could not compile `diffsitter` due to previous error
```

This problem came with 0.7.0, the previous version compiled fine.


**Environment:**
- Alpine Linux Edge
- rust 1.60.0

--- a/src/main.rs
+++ b/src/main.rs
@@ -6,6 +6,7 @@ mod input_processing;
 mod neg_idx_vec;
 mod parse;
 
+#[cfg(feature = "static-grammar-libs")]
 use crate::parse::supported_languages;
 use anyhow::Result;
 use clap::IntoApp;

